packages/client/src/components/app/charts/AreaChart.svelte

```diff
@@ -153,4 +168,4 @@
   }
 </script>
 
-<ApexChart {options} />
+<ApexChart {options} on:click{handleLineClick} />
```

Comment:
Does this handler ever do anything? I don't think the apex chart component broadcasts a click event. If it did, this would also then break because the params passed to the callback are not what `handleLineClick` expects. We should remove this `on:click` from the ApexChart component in every file here that it's been added to.